### PR TITLE
Add vat id format check in admin

### PR DESCRIPTION
**Pull request by JoshuaBehrens**
### 1. Why is this change necessary?

The vat id format is hidden on the database and nobody understands it.

### 2. What does this change do, exactly?

It adds a text input where users can enter data with a validation check about the user input against the format on the country.

image
image
image

### 3. Describe each step to reproduce the issue or behaviour.

1. Enable Vat ID validation for country
2. Customers get not accepted in registration
3. ╮(￣ω￣;)╭ 
4. Try to explain RegEx to client
5. Build something so the client admin users validate inputs themselves to verify customer requests
6. Have this now as a pull request

### 4. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/sw-settings-country-general.html.twig [hunk 1]
```diff
@@ -246,6 +246,33 @@
             />
             {% endblock %}
 
+            <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
+            {% block sw_settings_country_general_content_check_vat_id_format %}
```
Comment (keulinho) [inline src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/sw-settings-country-general.html.twig:250]: you should not add blocks to new admin js functionality

File: src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/index.js [hunk 2]
```diff
@@ -63,6 +64,25 @@ Component.register('sw-settings-country-general', {
 
         ...mapPropertyErrors('country', ['name']),
 
+        isTestVatValid() {
```
Comment (keulinho) [inline src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/index.js:67]: can you add a testcase for this?

File: src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/sw-settings-country-general.html.twig (hunk 1 above)
Comment (JoshuaBehrens) [reply src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/sw-settings-country-general.html.twig:250]: (｡•́︿•̀｡)

File: src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/index.js (hunk 2 above)
Comment (JoshuaBehrens) [reply src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/index.js:67]: I can try

**Comment by keulinho** [comment]
Hey @JoshuaBehrens again i'm closing this for inactivity, as always feel free to reopen if you can follow up again on the comments.

**Pull request closed without merging**